src/components/RichTextEditor/RichTextEditor.css

```diff
@@ -9,14 +9,25 @@
   --editor-container-width: 100%;
 }
 
+.text-editor-container {
+  display: flex;
+  flex-direction: var(--rich-text-editor-flex-direction, column);
+}
+
 .is-active {
   background: var(--button-background-color-is-active);
   color: var(--button-color-is-active);
 }
 
 .menu-bar-container {
+  align-items: var(--menu-bar-align-items, flex-start);
+  background-color: var(--menu-bar-background-color, --structure-color-white);
+  border-radius: var(--menu-bar-border-radius, 0);
   display: flex;
   flex-wrap: wrap;
+  height: var(--menu-bar-height, fit-content);
```

Comment:
@viniciusaa let's avoid using `fit-content` on the height or width. It's not widely supported yet and is known for causing some issues in other browsers.

Reply:
Any suggestions on what I should replace it with?

Reply:
@viniciusaa in this case, I would advise using the 100% value, if the intent is to make sure the component fits the entire width or height of whatever container it is nested under. Please make sure you double-check for side-effects though.